wikibase/1.30/bundle/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu as fetcher
+
+RUN apt-get update && \
+    apt-get install --yes --no-install-recommends unzip jq curl ca-certificates
+ADD download-extension.sh .
+RUN bash download-extension.sh OAuth
+RUN tar xzf OAuth.tar.gz && rm *.tar.gz
+
+FROM wikibase/wikibase:1.30
+COPY --from=fetcher /OAuth /var/www/html/extensions/OAuth
+COPY LocalSettings.php.template /LocalSettings.php.template.additional
```

Comment:
Perhaps this file should have a name specifying that it is from the bundle image?
LocalSettings.php.wikibase-bundle.template ? or something

Reply:
Sure